yoti_python_sdk/date_parser.py

```diff
@@ -9,4 +14,29 @@ def from_iso_format(string):
     if len(parts) != 3:
         raise ValueError
 
-    return datetime.date(parts[0], parts[1], parts[2])
+    return date(parts[0], parts[1], parts[2])
+
+
+def datetime_with_microsecond(string):
+    # Python2 does not have a way of parsing date formats.
+    # TODO Deprecate this once Python2 support is dropped.
```

Comment:
I prefer not having TODOs in code - we currently don't check for TODOs when releasing, but we do check that all issues in a particular release are done. Can we instead create an issue and add it to Python 3.0.0?
```suggestion
```